applications/FluidDynamicsApplication/custom_processes/mass_conservation_check_process.cpp

```diff
@@ -49,16 +51,18 @@ MassConservationCheckProcess::MassConservationCheckProcess(
     Parameters default_parameters( R"(
     {
         "model_part_name"                        : "default_model_part_name",
-        "mass_computation_frequency"             : 20,
+        "frequency_of_execution_in_time_steps"   : 20,
         "compare_to_initial_values"              : true,
-        "write_to_log_file"                      : true
+        "write_to_log_file"                      : true,
+        "log_file_name"                          : "mass_conservation.log"
     }  )" );
 
     rParameters.ValidateAndAssignDefaults(default_parameters);
 
-    mMassComputationFreq = rParameters["mass_computation_frequency"].GetInt();
+    mMassComputationFreq = rParameters["frequency_of_execution_in_time_steps"].GetInt();
```

Comment:
just a hint: you could use here the other constructor. (what you did is ok, just take this as a hint for the future)